openapi/src/parts/workspace.yaml

```diff
@@ -253,6 +253,11 @@ components:
         destinationWorkspaceId:
           type: string
           format: uuid
+        destinationUserFacingId:
+          description: |
+            Human-settable, mutable id. Must have 3-63 characters, contain lowercase letters, numbers, dashes, or
+            underscores, and start with lowercase letter. Required.
```

Comment:
This is an output property so you don't need to mark it required.

Reply:
Done